### PR TITLE
floating point arithmetic in python

### PR DESCRIPTION
**Pull request by p-rit**
#630 solved.

### CODE REVIEW

File: float_devide.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# to perform float devision in range .9999e49to .1000e-50
```
Comment (jainaman224) [inline float_devide.py]: There is a typo in file name.

File: float_sub.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# to perform float substraction in range .9999e49to .1000e-50
```
Comment (jainaman224) [inline float_sub.py]: Please use float_subtract.py as file name.

**Comment by jainaman224** [comment]
Please add all files to a folder.

**Comment by p-rit** [comment]
updated files as per the review

**Comment by jainaman224** [comment]
I am really sorry. I have done some stupidity. I need your help to fix this.

**Pull request closed without merging**